docs/upscale.rst

```diff
@@ -10,7 +10,7 @@ This filter tells thumbor to upscale your images. This only makes sense with
 "fit-in" or "adaptive-fit-in".
 
 This means that if an original image is 300px width by 200px height and you
-ask for a 600x500 image, the filter will resize it to 600x400.
+ask for a 600x500 image, the filter will resize it to 600x500.
```

Comment:
Are you sure it's right? It suppose to fit image into a box of 600x500 keeping aspect ratio.